File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0002 c++/ludo.cpp
#include <iostream>

int main(){
    int test;
    std::cin >> test;
    while(test > 0){
        int x;
        std::cin >> x;
        if(x == 6){
            std::cout << "YES" << std::endl;
        }else{
            std::cout << "NO" << std::endl;
        }
        test = test - 1;
    }
    return 0;
}

File: 0001 Code Chef/0023 Chef Plays Ludo/implementation/0010 nodejs/ludo.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 1;

    rl.on('line',(input)=>{
        let x = parseInt(input);
        if(x == 6){
            console.log("YES");
        }else{
            console.log("NO");
        }
        if(count == test){
            rl.close();
        }else{
            count = count + 1;
        }
    })
})